OHIF/Viewers
# feat: Add CS3D WSI and Video Viewports and add annotation navigation for MPR

### Context

Video and WSI images now have a video viewport.  This PR adds the ability to display video and wsi using the CS3D viewports instead of the OHIF ones, allowing things like annotation and segmentations to be shared.

### Changes & Results

Use video viewport for basic-test mode

- pan/zoom tools for video
- tracked annotation tools for video
- "CINE" playback using video play, with rate being relative to recording rate.
- demographics overlay

Use WSI viewport for basic-test mode

- pan/zoom tools
- needs WSI viewport loading PR to actually load images (when this is linked in/merged, the WSI testing can proceed)

Switch measurement navigation to use view references

Add tracked changes for wsi and video viewports.

MPR viewports now support annotation navigation (using setViewReference).

### Testing

Against the ohif back end, use the studies:
Horse - for video
Any SM study for microscopy
Any CT study for volume navigation

Check that video and microscopy viewports can have annotations added.
Check that on the horse study, you can add an annotation to the stack and to the video viewport, and that clicking on them in the tracked changes navigates to the specified display set and image position.
Check that on the horse study you can play video using the CINE controls
Check that on video and microscopy you can zoom/pan
Check that you can window level on all 4 viewport types

Test MPR navigation with:
1. Turn on MPR (basic-test mode is easiest as you can hard turn off MPR with a button)
2. Add an annotation to all three viewports
3. Turn off MPR
4. Click on any of the annotations and have the single viewport navigate to the specific view in the MPR AND show the correct annotation

Note how the volume viewport gets recreated if MPR is off, however it does NOT show the 3x1 MPR hanging protocol but merely shows the single viewport.

### Checklist

#### PR



- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates



- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
